docs/content/latest/quick-start/build-apps/java/ysql-ebeans.md

```diff
@@ -0,0 +1,369 @@
+aliases:
+  - /develop/client-drivers/java/
+  - /latest/develop/client-drivers/java/
+  - /latest/develop/build-apps/java/
+  - /latest/quick-start/build-apps/java/
+  - /latest/integrations/jdbc-driver
+menu:
+  latest:
+    parent: build-apps
+    name: Java
+    identifier: java-7
+    weight: 550
+type: page
+isTocNested: true
+showAsideToc: true
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+
+  <li >
+    <a href="../ysql-yb-jdbc/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - YB - JDBC
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-jdbc/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - JDBC
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-jdbc-ssl/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - JDBC SSL/TLS
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-spring-data/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - Spring Data JPA
+    </a>
+  </li>
+   <li>
+    <a href="../ysql-ebeans/" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - Ebeans
+    </a>
+  </li>
+  <li>
+    <a href="../ycql/" class="nav-link">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL
+    </a>
+  </li>
+  <li>
+    <a href="../ycql-4.6/" class="nav-link">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL (4.6)
+    </a>
+  </li>
+</ul>
+
+The following tutorial implements an ORM example using [Ebeans](https://ebean.io/docs/), an ORM in Java, that implements a simple REST API server. The scenario is that of an e-commerce application where the database access is managed through [Play framework](https://www.playframework.com/documentation/2.8.x/api/java/index.html); Play uses [Akka](https://doc.akka.io/docs/akka/current/typed/guide/introduction.html) internally and exposes Akka Streams and actors in Websockets and other streaming HTTP responses. It includes the following tables:
+
+- `users` table — the users of the e-commerce site
+- `products` table — the products being sold
+- `orders` table — the orders placed by the users
+- `orderline` table — each line item of an order
+
+The source for the above application can be found in the [Using ORMs with YugabyteDB](https://github.com/yugabyte/orm-examples/tree/master/java/ebeans) repository.
+
+## Prerequisites
+
+This tutorial assumes that you have:
+
+- YugabyteDB up and running. If you are new to YugabyteDB, follow the steps in [Quick start](../../../../quick-start/) to have YugabyteDB up and running in minutes.
+- Java Development Kit (JDK) 1.8 is installed. JDK installers for Linux and macOS can be downloaded from [OpenJDK](http://jdk.java.net/), [AdoptOpenJDK](https://adoptopenjdk.net/), or [Azul Systems](https://www.azul.com/downloads/zulu-community/).
+- [sbt](https://www.scala-sbt.org/1.x/docs/) is installed.
```

Comment:
Add info about sbt version; build.properties doesn't appear to be in the repository. perhaps it should be, with 'sbt.version = 1.2.8' added, then maybe it would just work

Reply:
I agree with @ddhodge . @priyanshi-yb If the build.properties file is not created before running sbt compile, the compile will create a build.properties file under the project folder. With the current inconsistent behaviour  of sbt versions, if would be better to create a build.properties file with the version 1.2.8 in the repository , rather than letting the sbt compile add the system version which may not work. @priyanshi-yb , let us know if this is right and make the change in the repository as well.

Reply:
Agree, added the steps for that in the docs, will change the repo's readme with appropriate changes in some time.